src/mlpack/methods/ann/layer/noisylinear.hpp

```diff
@@ -133,6 +133,8 @@ class NoisyLinear
   //! Modify the bias weights of the layer.
   arma::mat& Bias() { return bias; }
 
+  //! Get size of weights.
+  size_t WeightSize() const { return (outSize * inSize + outSize) * 2; }
```

Comment:
```suggestion
  size_t WeightSize() const { return (outSize * inSize + outSize) * 2; }

```
A newline here would be good.
It's just a small style fix to retain the general style format across the files.